adb_graphics/figure_builders.py

```diff
@@ -52,9 +52,10 @@ def add_obs_panel(ax, model_name, obs_file, proj_info, short_name, tile):
     dm.draw(show=True)
 
 def parallel_maps(cla, fhr, ds, level, model, spec, variable, workdir,
-                  tile='full'):
+                  tile='full', ds2=None):
 
     # pylint: disable=too-many-arguments,too-many-locals
+    # pylint: disable=too-many-branches,too-many-statements
```

Comment:
This function could likely use some refactoring given this growing list of disabled linting concerns.